force-app/main/default/classes/RD2_OpportunityEvaluationService.cls

```diff
@@ -449,6 +449,7 @@ public inherited sharing class RD2_OpportunityEvaluationService {
             //when Opportunities were evaluated so no need to redo the work
             if (rdRecord
                 .reviseYearValues(yearValueProcessor)
+                .storeAndClearChangeType()
```

Comment:
Notes: This may affect performance because the DML update over here is separate from the original trigger update.